docs/reference/catalogbase-reference.md

```diff
@@ -113,24 +89,20 @@ The `place_order()` function allows you to place a catalog or tasking order.
 ```python
 place_order(
     order_parameters,
-    track_status=False,
-    report_time=120,
+    track_status,
+    report_time,
 )
 ```
 
-<h5> Arguments </h5>
-
-| Name               | Type                | Description                                                                                                                                                                                |
-| ------------------ | ------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
-| `order_parameters` | `Union[dict, None]` | Parameters with which to place an order.                                                                                                                                                   |
-| `track_status`     | `bool`              | Determines when to return order data:<ul><li>`True`: return order data only when the order status changes to `FULFILLED` or `FAILED`.</li><li>`False`: return order data immediately.</li></ul> |
-| `report_time`      | `int`               | Use if `track_status=True`.<br/><br/>The time interval for querying whether the order status has changed to `FULFILLED` or `FAILED`, in seconds.                                                |
+The returned format is `Order`.
 
-<h5> Returns </h5>
+<h5> Arguments </h5>
 
-| Type    | Description |
-| ------- | ----------- |
-| `Order` | Order data. |
+| Argument           | Overview                                                                                                                                                                                                                                      |
+| ------------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `order_parameters` | **Union[dict, None]**<br/>Parameters with which to place an order.                                                                                                                                                                            |
+| `track_status`     | **bool**<br/>Determines when to return order data:</p><ul><li>`True`: return order data only when the order status changes to `FULFILLED` or `FAILED`.</li><li>`False`: return order data immediately.</li></ul>The default value is `False`. |
+| `report_time`      | **int**<br/>Use if `track_status=True`.<br/><br/>The time interval for querying whether the order status has changed to `FULFILLED` or `FAILED`, in seconds. The default value is `120`.                                                      |
```

Comment:
```suggestion
| `report_time`      | **int**<br/>Use if `track_status=True`.<br/><br/>The time interval (in seconds) for querying whether the order status has changed to `FULFILLED` or `FAILED`. The default value is `120`.                                                      |
```

Reply:
Why?

Reply:
But that's the whole definition of the interval. Also, this is the format that we use: `This is this thing, in units.` So I'm not sure about that change (and we try to never use [parentheses](https://developers.google.com/style/parentheses?hl=en)).